mail_restrict_follower_selection/utils.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+def _id_get(env, id_str, raise_if_not_found=True):
+    """Have a more secure ref function for use with safe_eval.
+
+    Returning only the ID of the record.
+
+    """
+    return env.ref(id_str, raise_if_not_found).id
```

Comment:
This is not going to work if not found:
```
>>> env.ref("not.exists", False).id
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'id'
```
Your simplest option might be to just drop `raise_if_not_found` from the method signature.

Reply:
I removed this parameter to make things easier, thank you.

Commits squashed as well so the PR is ready to merge.